client/src/components/portfolio/positionTable.jsx

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+import PropTypes from 'prop-types'
+import Box from '@mui/material/Box';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TablePagination from '@mui/material/TablePagination';
+import TableRow from '@mui/material/TableRow';
+import TableSortLabel from '@mui/material/TableSortLabel';
+import { visuallyHidden } from '@mui/utils';
+import mockDataP from '../../../../mockDataP.js';
+
+const headCells = [
+  {id: 'symbol', numeric: false, label: 'Symbol'},
+  {id: 'lastPrice', numeric: true, label: 'Last Price'},
+  {id: 'gainLossPer', numeric: true, label: '% Total Gain/Loss'},
+  {id: 'gainLossDol', numeric: true, label: '$ Total Gain/Loss'},
+  {id: 'accountPer', numeric: true, label: '% of Account'},
+  {id: 'qty', numeric: true, label: 'Quantity'},
+  {id: 'avgCost', numeric: true, label: 'Average Cost Basis'}
+];
+
+const rows = mockDataP.mockPortfolioData;
+
+function descendingComparator(a, b, orderBy) {
+  if (b[orderBy] < a[orderBy]) {
+    return -1;
+  }
+  if (b[orderBy] > a[orderBy]) {
+    return 1;
+  }
+  return 0;
+};
+
+function getComparator(order, orderBy) {
+  return order === 'desc'
+    ? (a, b) => descendingComparator(a, b, orderBy)
+    : (a, b) => -descendingComparator(a, b, orderBy);
+};
+
+function stableSort(array, comparator) {
+  const stabilizedThis = array.map((el, index) => [el, index]);
+  stabilizedThis.sort((a, b) => {
+    const order = comparator(a[0], b[0]);
+    if (order !== 0) {
+      return order;
+    }
+    return a[1] - b[1];
+  });
+  return stabilizedThis.map((el) => el[0]);
+}
+
+function PositionTableHead(props) {
+  const {order, orderBy, onRequestSort } = props;
+  const createSortHandler = (property) => (event) => {
+    onRequestSort(event, property);
+  };
+  return (
+    <TableHead>
+      <TableRow>
```

Comment:
I learned something new about MUI, it makes no sense that a TableRow would go inside a TableHead but yeah, that's how it works